apps/studio/components/interfaces/Home/ServiceStatus.tsx

```diff
@@ -113,13 +132,52 @@ const ServiceStatus = () => {
   const isLoadingChecks = services.some((service) => service.isLoading)
   const allServicesOperational = services.every((service) => service.isSuccess)
 
+  // If the project is less than 5 minutes old, and status is not operational, then it's likely the service is still starting up
+  const isProjectNew =
+    dayjs.utc().diff(dayjs.utc(project?.inserted_at), 'minute') < SERVICE_STATUS_THRESHOLD
+
+  const StatusMessage = ({ isLoading, isSuccess }: { isLoading: boolean; isSuccess: boolean }) => {
+    if (isLoading) return 'Checking status'
+    if (isProjectNew) return 'Coming up...'
+    if (isSuccess) return 'No issues'
+    return 'Unable to connect'
+  }
+
+  const StatusIcon = ({ isLoading, isSuccess }: { isLoading: boolean; isSuccess: boolean }) => {
+    if (isLoading || isProjectNew) return <Loader2 size={14} className="animate-spin" />
+    if (isSuccess) return <CheckCircle2 className="text-brand" size={18} strokeWidth={1.5} />
+    return <AlertTriangle className="text-warning" size={18} strokeWidth={1.5} />
+  }
```

Comment:
Can these be defined outside of the render method? I find there are often random bugs when defining other components inside components!

Reply:
oh yes gotcha! overlooked this one 🙏